frontend_tests/node_tests/narrow_state.js

```diff
@@ -23,6 +24,12 @@ function set_filter(operators) {
 
     assert(!narrow_state.is_for_stream_id(test_stream.stream_id));
 
+    var bad_stream_id = 1000000;
+    global.blueslip.error = function (msg) {
```

Comment:
This seems like a good place to introduce another testing practice that we do. Simply mocking `blueslip.error` here doesn't mean that `global.blueslip.error` is actually called. So, let's make sure it's called by adding `var called = false;` outside the function. And then inside the function, add `called = true;`. Finally, after the function, we want to `assert(called);` to make sure that `blueslip.error` is actually called.